### ISSUE TITLE
Add description/name option for global hooks

### ISSUE THREAD

**Issue by dietergeerts** [cucumber/cucumber-js#1079]
I searched for it, but couldn't find out how to do this, even from the source code.
So basically, we now have this kind of output:

```
1) Scenario: Create meeting # features\MI0465-203-create-meeting.feature:10
   √ Before # features\step_definitions\hooks.js:31
   √ Before # features\step_definitions\hooks.js:43
   √ Given Actor Manager # features\step_definitions\actor.js:10
   √ When I wish to create a Meeting # features\step_definitions\meetings.js:13
...
   × Then The Meeting has the type Express # features\step_definitions\meeting-form.js:52
       NoSuchElementError: No element found using locator: By(css selector, *[id="meeting-type-name"])
...
```
 As there can be more than 1 global (before) hook ran during a scenario, it would be handy to know which ones where run. Especially because we use tagged global hooks, so depending on the tags, some will run, others won't. If a test fails, we would like to see what has been run.

Is this available right now? And if not, can this be added?

**Comment by charlierudolph** [comment]
Okay. This isn't currently available but we could add it as an option alongside the timeout for all hooks

**Comment by davidjgoss** [comment]
Just calling out that this is underway via https://github.com/cucumber/common/issues/1139

**Comment by davidjgoss** [comment]
This has now been released in [v8.1.0](https://github.com/cucumber/cucumber-js/releases/tag/v8.1.0) on npm

https://www.npmjs.com/package/@cucumber/cucumber
